src/pkg/exporter/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"github.com/vmware-tanzu/cloud-native-security-inspector/src/lib/log"
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/forward", mainHandler)
+	listenPort := 6780
```

Comment:
Why should this port be hard-coded?  Do we plan to make this value read from a configuration?

Reply:
I think it's pretty like 3306 for mysql, just a convention.

This is an internal component of our software, so I think we can set whatever we want. It will not conflct with others because it is in the port space of the pod of data exporter, having no conflict with others.